cl/disclosures/tests.py

```diff
@@ -217,3 +221,20 @@ def test_filter_related_object(self) -> None:
 
         r = self.client.get(self.path, self.q)
         self.assertEqual(r.json()["count"], 1, msg="Wrong disclosure count")
+
+
+class DisclosureReactLoadTest(BaseSeleniumTest):
+    @timeout_decorator.timeout(SELENIUM_TIMEOUT)
+    @override_flag("financial-disclosures", active=True)
```

Comment:
Why not yank the waffle instead of building on it? Seems better at this point?

Reply:
hm... I thought it might be useful to have a template for future waffle tests.  but I can certainly yank it'